js/screens/HomeScreen.js

```diff
@@ -78,10 +82,34 @@ class HomeScreen extends React.Component {
         }
       });
     } else {
+      this.donateShortcut(site);
       this.props.screenProps.openUrl(`${site.url}`);
     }
   }
 
+  donateShortcut(site) {
+    if (Platform.OS !== 'ios') {
```

Comment:
Do other Apple platforms identify themselves as "ios"? I'm thinking mostly of iPadOS, but Siri shortcut support on macOS could be useful in the future

Reply:
Good question, this is a React Native module, and under [Platform](https://reactnative.dev/docs/platform#os) it only lists android and ios. 

I believe other Platform checks work fine on macOS, but I will look into this further anyway, due to an auth issue with the app in macOS.